src/main/minio.js

```diff
@@ -924,12 +925,12 @@ export class Client {
         // if there was a previous incomplete upload, fetch all its uploaded parts info
         if (uploadId) return this.listParts(bucketName, objectName, uploadId,  (e, etags) =>  cb(e, uploadId, etags))
         // there was no previous upload, initiate a new one
-        this.initiateNewMultipartUpload(bucketName, objectName, '', (e, uploadId) => cb(e, uploadId, []))
+        this.initiateNewMultipartUpload(bucketName, objectName, metaData, (e, uploadId) => cb(e, uploadId, []))
```

Comment:
Should this argument be an empty javascript object , {}, instead of metadata, as in the old code content-type was initialized to an empty string there.